core/geometry/src/test/geometry3d/Point3dVector3d.test.ts

```diff
@@ -214,6 +214,90 @@ describe("Point3d.setFrom", () => {
   });
 });
 
+describe("Vector3d", () => {
+  it("hello", () => {
+    const ck = new bsiChecker.Checker();
+    const pointA = Point3d.create(1, 2, 5);
+    const pointB = Point3d.create(4, 2, 9);
+    const q = 3.902;
+    const vectorABq = pointA.scaledVectorTo(pointB, q);
+    const vectorAB = pointA.vectorTo(pointB);
+    ck.testParallel(vectorAB, vectorABq, "parallel vectors");
+    ck.testCoordinate(q * vectorAB.magnitude(), vectorABq.magnitude(), "enforced magnitude");
+
+    const vectorABxyz = Vector3d.createStartEndXYZXYZ(pointA.x, pointA.y, pointA.z, pointB.x, pointB.y, pointB.z);
+    ck.testVector3d(vectorAB, vectorABxyz);
+
+    ck.checkpoint("Vector3d.hello");
+    expect(ck.getNumErrors()).equals(0);
+  });
+
+  it("CrossProducts", () => {
+    const ck = new bsiChecker.Checker();
+    const vectorA = Vector3d.create(-4, 4, 2);
+    const unitZ = Vector3d.unitZ();
+    const vectorB = Vector3d.create(0.3, 9.1, -2);
+    const pointB0 = Point3d.create(3, 2, 8);
+    const pointB1 = pointB0.plus(vectorB);
+    ck.testCoordinate(vectorA.crossProductXY(vectorB), vectorA.tripleProduct(vectorB, unitZ), "crossProductXY");
+    ck.testCoordinate(vectorA.crossProductStartEndXY(pointB0, pointB1), vectorA.tripleProduct(vectorB, unitZ), "crossProductXY");
```

Comment:
Nice!

Reply:
these were few tests in an isolated file so I moved them here and deleted that file